From 1a978048363e56bab9edba3ade404df96bc40a80 Mon Sep 17 00:00:00 2001
From: Sai Krishna Potthuri <lakshmi.sai.krishna.potthuri@xilinx.com>
Date: Thu, 25 Jun 2020 17:36:21 +0530
Subject: [PATCH 1403/1852] mtd: cadence-quadspi: Fixed UNUSED_VALUE warning

commit 5c9eb0f96f4a3108c5c1b7b5d9ed9f56933e7625 from
https://github.com/Xilinx/linux-xlnx.git

This patch removed assigning 'ret' variable to 1 as this value is
overwritten before it can be used.

Addresses-Coverity: "Unused value (UNUSED_VALUE)"
Signed-off-by: Sai Krishna Potthuri <lakshmi.sai.krishna.potthuri@xilinx.com>
Signed-off-by: Radhey Shyam Pandey <radhey.shyam.pandey@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/mtd/spi-nor/cadence-quadspi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mtd/spi-nor/cadence-quadspi.c b/drivers/mtd/spi-nor/cadence-quadspi.c
index 13be70b428dd..11df25387300 100644
--- a/drivers/mtd/spi-nor/cadence-quadspi.c
+++ b/drivers/mtd/spi-nor/cadence-quadspi.c
@@ -1555,7 +1555,7 @@ static int cqspi_setdlldelay(struct spi_nor *nor)
 	struct cqspi_st *cqspi = f_pdata->cqspi;
 	int i;
 	u8 j;
-	int ret = 1;
+	int ret;
 	u8 id[CQSPI_READ_ID_LEN];
 	bool rxtapfound = false;
 	u8 min_rxtap = 0;
-- 
2.31.1

